concent-builder/configure.yml

```diff
@@ -111,3 +111,10 @@
             state: directory
             owner: "{{ shared_user }}"
             group: "{{ shared_user }}"
+
+        - name: Upload configuration for kubectl
+          copy:
+            src:   "{{ local_secret_dir }}/cloud/config"
+            dest:  "/home/{{ shared_user }}/.kube/config"
+            owner: "{{ shared_user }}"
+            group: "{{ shared_user }}"
```

Comment:
One problem I see: we're doing this only for `shared_user`. Don't we need to do this on the account of every user who might run the playbook? Or is everyone still deploying from this user account?

Reply:
Yes with still use one user account for deploying. https://github.com/golemfactory/concent-deployment/pull/223